### ISSUE TITLE
Focus is forced to the website results regardless of the dictionaries order in the results display.

### ISSUE THREAD

**Issue by ENigma885** [goldendict/goldendict#272]
**Problem:
- When looking up a word, the website result is focused as soon as it's been loaded regardless of the dictionaries predefined order.

**Environment:
- GoldenDict Bleeding-edge 1.0.1-574-g7fa2943. (Compiled from source as directed @ http://goldendict.org/buildfromgit.php)
- Ubuntu 12.10

**How to reproduce:
1- Add an online dictionary, in my case it's http://www.wordreference.com/enar/%GDWORD%, but tried with others also.
2- Activate it by checking the box of the "Enabled" column.
3- Look up any word.
4- Results are displayed in the predefined order of dictionaries made by the user. Once the online result has been fully loaded, the focus is instantly changed to that result regardless of the order of dictionaries, which caused real distraction.

**Comment by xiaoyifang** [comment]
> I don't want to look like Mr. Obvious here because of my lack of the tiniest developing and programming skills :D, but is there a way to make webkit ignore these focus requests as long as they are not of any use in a multi-dictionaries program after all?

yes ,there is a way. this has been fixed in goldendict-ng. have a try if you like.

**Comment by vedgy** [comment]
> yes ,there is a way. this has been fixed in goldendict-ng. have a try if you like.

Users: beware of goldendict-ng's downsides (#1616) and of deliberate one-sided advertisement on the part of goldendict-ng's developers (#1618).

**Comment by vedgy** [comment]
> > has been fixed in goldendict-ng
> 
> But the ng seems more like a joke made by pigbrothers.

@nonwill, please stop posting these pointless comments. Just downvote the offending comments by @xiaoyifang. And if no one else does it before, politely point users to #1616 and #1618.

**Comment by nonwill** [comment]
OK
